app/(tabs)/ProfileView/BookmarkedSpotsView.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { View, Text, TouchableOpacity, StyleSheet, FlatList, Image } from 'react-native';
+import { AntDesign } from '@expo/vector-icons';
+import { NavigationProp, useNavigation } from '@react-navigation/native';
+
+interface BookmarkedSpot {
```

Comment:
Check out the useContext() function when you are finished to import the bookmarkedRestaurants variable that holds a list of Restaurant objects (or will do when the API calls are done)